instaclone/src/App.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import uuid from 'uuid';
+import "./App.css";
+import dummyData from './dummy-data';
+import SearchBar from './components/SearchBar/SearchBar';
+import PostContainer from './components/PostContainer/PostContainer';
+
+function App() {
+  return (
+    <div className='App'>
+      <SearchBar />
+      {dummyData.map(data => {
+        console.log(data);
+        return <PostContainer key={uuid()} data={data} />;
```

Comment:
What is this uuid? There's no need. Set the key to something unique in the data file.